velox/type/Variant.h

```diff
@@ -422,6 +419,13 @@ class variant {
     }
   }
 
+  // support construction from StringView as well as StringPiece
```

Comment:
nit: we usually use full sentences in comments (capitalization and punctuation).

Reply:
Thanks, I'll rewrite it